test/struct/TestClient.js

```diff
@@ -1,10 +1,11 @@
 const { AkairoClient, CommandHandler, InhibitorHandler, ListenerHandler, SQLiteProvider } = require('../../src/index');
 const sqlite = require('sqlite');
+const sqlite3 = require('sqlite3');
 
 class TestClient extends AkairoClient {
-    constructor() {
+    constructor(ownerID) {
         super({
-            ownerID: '123992700587343872'
+            ownerID: ownerID
```

Comment:
This can be shortened down to:
```js
super({ ownerID })
```

Reply:
Thanks for the advice, made this change.